moveit_commander/test/python_moveit_commander_ns.py

```diff
@@ -0,0 +1,69 @@
+#!/usr/bin/env python
+
+import unittest
+import numpy as np
+import rospy
+import rostest
+import os
+
+from moveit_commander import RobotCommander
+
+
+class PythonMoveitCommanderNsTest(unittest.TestCase):
+    PLANNING_GROUP = "manipulator"
+    PLANNING_NS = "test_ns/"
+
+    @classmethod
+    def setUpClass(self):
+        self.commander = RobotCommander("%srobot_description"%self.PLANNING_NS, self.PLANNING_NS)
+        self.group = self.commander.get_group(self.PLANNING_GROUP)
+
+    @classmethod
+    def tearDown(self):
+        pass
+
+    def check_target_setting(self, expect, *args):
```

Comment:
rename something like `test__group__set_joint_value_target` to make it easier to understand what this test does